samples/keycloak-pkce-cmdline-sample/README.md

```diff
@@ -0,0 +1,36 @@
+# Instructions for the Keycloak OAuth2 with PKCE Command-Line Sample
+
+## Browse Online
+
+[Browse Source][browse-source], or main file [DailyMotionSample.java][main-source].
+
+## Command-Line Instructions
+
+**Prerequisites:** install [Java 6 or higher][install-java], [git][install-git], and
```

Comment:
Java 7

Reply:
Ok, this is copied from the dailymotion sample, that says Java 6 and as per my reply to your comment below, I've removed the config block from the pom that specified a java version explicitly. So should I still change this to 7?

I'll see if I can download Java 6 somewhere and see if everything seems to work if I use that.

Reply:
No, the daily motion sample should say 7. Those docs might just be out of date.

Reply:
ahh... understood, will change the text in the daily motion sample as well.